### ISSUE TITLE
Custom Model Creation Interface

### ISSUE THREAD

**Issue by DDunmore** [smeighan/xLights#2529]
**Is your feature request related to a problem? Please describe.**
Creating Custom Models although the interface has come a long way to how it originally was is still quite clunky/painful expecially when creating larger detailed props with if you up the resolution of the grid it becomes hard to see the overall image unless you have a huge monitor, and you have to zoom right in to be able to click onto pixels..

**Describe the solution you'd like**
Would it be possible to implement the model creation process to be similar to how https://www.lightshowhub.com/tools/xlights-custom-model-builder works.  Using this tool although maybe not perfect, seems to be a lot faster/easier to create/edit models and the creation of faces/submodels.   
It has the abiltity to size up or down pixels just in reference to a background image, then you can adjust the grid size at the end to get the resolution to fit into xlights grid nicely.  It just seems are more polished abiltiy to create a custom model that would be nice to have a similar setup practice within xlights not online.

**Comment by AzGilrock** [comment]
Yes his tool is nice.  It would be nice if people would create tools inside xLights instead of everyone wanting to create their own 3rd party tools.  Now that it exists I can't see spending a ton of my hours for free to duplicate what is already out there for people to use.  But there are many developers so maybe you will get lucky and someone will be interested.
